client/python/gradio_client/serializing.py

```diff
@@ -353,6 +353,113 @@ def deserialize(
             )
 
 
+class RichtextboxSerializable(Serializable):
```

Comment:
do we need a new serializeable type? Is JSON serializable not sufficient (its enough for Chatbot, HighlightedText, etc.)

Reply:
Yeah we do, `JSONSerializable` can't handle files which have this ugly dictionary structure. This will get removed in 4.0 so I wouldn't worry too much about it

Reply:
That being said, tiny nit here regarding capitalization:

`RichtextboxSerializable` should be `RichTextboxSerializable`

@dawoodkhan82